client/packages/common/src/ui/components/navigation/Tabs/ModalTab.tsx

```diff
@@ -15,7 +15,7 @@ const StyledTabPanel = styled(TabPanel)({
 
 const StyledTabContainer = styled(Box)(({ theme }) => ({
   borderColor: theme.palette.divider,
-  flexDirection: 'row',
+  flexDirection: 'column',
```

Comment:
I was nervous about the impact on other uses of the ModalTab.. but is fine so I'll calm down.